sdk/security/keyvault/azadmin/backup/autorest.md

```diff
@@ -0,0 +1,90 @@
+## Go
+
+```yaml
+clear-output-folder: false
+export-clients: true
+go: true
+input-file: 
+    - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.4/backuprestore.json
+license-header: MICROSOFT_MIT_NO_VERSION
+openapi-type: "data-plane"
+output-folder: ../backup
+override-client-name: Client
+security: "AADToken"
+security-scopes: "https://vault.azure.net/.default"
+use: "@autorest/go@4.0.0-preview.44"
```

Comment:
At some point you'll need to move to the `preview.46` code generator.  This might require some refactoring of your hand-written code, so feel free to punt this for now.

Reply:
will update in next release

Reply:
fixed in 9848f66